plugin/converter/jsonnet/jsonnet.go

```diff
@@ -125,7 +127,17 @@ func Parse(req *core.ConvertArgs, fileService core.FileService, limit int, templ
 	if len(templateData) != 0 {
 		for k, v := range templateData {
 			key := fmt.Sprintf("input." + k)
-			val := fmt.Sprint(v)
+			var val string
+			switch reflect.TypeOf(v).Kind() {
+			case reflect.Slice, reflect.Map:
+				data, err := yaml.Marshal(v)
```

Comment:
If we had to choose between json and yaml, I would probably serialize to json for consistency reasons, since this is how we serialize plugin parameters that are complex slice or map values.  But otherwise looks good to me.